sagemaker_processing/fairness_and_explainability/text_explainability_sagemaker_algorithm/byo_blazingtext_model_hosting.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    model_name = "blazing-text-byo-model-clarify-demo"
</pre><p>Please add a timestamp suffix to the model name, so multiple unique models will be created if the example is run multiple times.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Fixed.